### PR TITLE
add cn doc for paddle.vision.transforms.erase and paddle.vision.transforms.RandomErasing

### PR DESCRIPTION
**Pull request by zoooo0820**
add cn docs for two new API `paddle.vision.transforms.erase` and `paddle.vision.transforms.RandomErasing`

The API PR is [#42280](https://github.com/PaddlePaddle/Paddle/pull/42280)

### CODE REVIEW

**Review by qingqing01 — APPROVED** [review]

File: docs/api/paddle/vision/transforms/erase_cn.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - j (int) - 擦除区域左上角点的横坐标。
+    - h (int) - 擦除区域的高。
+    - w (int) - 擦除区域的宽。
+    - v (paddle.Tensor|np.array) - 用于替换擦除区域中像素的值。当输入为np.array或者PIL.Image类型时，需要为np.array类型。
+    - inplace (bool, 可选) - 该变换是否在原地操作。默认值：False。
+
+返回
+:::::::::
+
+    ``paddle.Tensor 或 numpy.array 或PIL.Image``，擦除后的图像，类型与输入图像的类型一致。
```
Comment (dingjiaweiww) [inline docs/api/paddle/vision/transforms/erase_cn.rst]: 或在引号外面

File: docs/api/paddle/vision/transforms/erase_cn.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+代码示例
+:::::::::
+
+.. code-block:: python
+
+    import paddle
+                
+    fake_img = paddle.randn((3, 10, 10)).astype(paddle.float32)
+    values = paddle.zeros((1,1,1), dtype=paddle.float32)
+    result = paddle.vision.transforms.erase(fake_img, 4, 4, 3, 3, values)
```
Comment (dingjiaweiww) [inline docs/api/paddle/vision/transforms/erase_cn.rst]: 示例代码没有给出打印的结果

**Review by dingjiaweiww — APPROVED** [review]

File: docs/api/paddle/vision/transforms/erase_cn.rst (hunk 1 above)
Comment (zoooo0820) [reply docs/api/paddle/vision/transforms/erase_cn.rst]: Done, thanks

File: docs/api/paddle/vision/transforms/erase_cn.rst (hunk 2 above)
Comment (zoooo0820) [reply docs/api/paddle/vision/transforms/erase_cn.rst]: Done, 减小了用例shape并给出了打印结果

**Review by dingjiaweiww — APPROVED** [review]
LGTM

**Pull request merged**